### ISSUE TITLE
todo test: when fieldsKey/filesKey not string

### ISSUE THREAD

**Issue by tunnckoCore** [helapkg/hela#10]
``` js
if (typeof opts.fieldsKey !== 'string') {
  copy = copy.fields;
}
```

@pepkin88 also need to check if `filesKey` is string.. xD

Mainly, the idea behind this checks and the `false` concept was, if option is set to false, then pass the body, directly to the `this.request` object (as you mention `this.request.files`)

**Comment by tunnckoCore** [comment]
I'll publish it, but will TBC with this.

**Comment by tunnckoCore** [comment]
@pepkin88 published as `v1.0.13`

**Comment by pepkin88** [comment]
Because in https://github.com/tunnckoCore/koa-better-body/commit/73d63a0da278a5b0c8fdc2b735e6b4ab4bbdcc35 the `filesKey` was ignored if it was not string, I didn't extended body with `files`. But maybe it's a good idea to left such option, just in case.
Also, maybe instead of checking if something is a string or not, it should be checked it is equal to `false` or not. That way symbols can be used.
